src/dialogs/edittagdialog.cpp

```diff
@@ -112,13 +113,15 @@ EditTagDialog::EditTagDialog(Application *app, QWidget *parent)
       tag_fetcher_(new TagFetcher(app->network(), this)),
       results_dialog_(new TrackSelectionDialog(this)),
 #endif
+      lyrics_fetcher_(new LyricsFetcher(app->lyrics_providers(), this)),
       image_no_cover_thumbnail_(ImageUtils::GenerateNoCoverImage(QSize(128, 128), devicePixelRatioF())),
       loading_(false),
       ignore_edits_(false),
       summary_cover_art_id_(-1),
       tags_cover_art_id_(-1),
       cover_art_is_set_(false),
-      save_tag_pending_(0) {
+      save_tag_pending_(0),
+      lyrics_id_(-1) {
```

Comment:
```suggestion
      lyrics_fetch_id_(-1) {
```

Reply:
I copied that name from context/contexview.cpp, in which a similar variable does the same thing. I think it should remain the same for consistency (or be changed in both)